web2.0/src/Components/Current/CurrSwell.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from 'react';
+import WeatherCard from '../weather-card/WeatherCard';
+
+const CurrSwell = () => {
+  const [currentData, setCurrentData] = useState();
+
+  const getData = async () => {
+    const response = await fetch('http://localhost:8000/swell');
+    const data = await response.json();
+    setCurrentData(data);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+  return (
+    <div>
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].waveHeight.dataLabel}
+        weatherData={currentData?.[0].waveHeight.data}
+        extraData={currentData?.[0].waveHeight.unit}
+        className={currentData?.[0].waveHeight.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].swellHeight.dataLabel}
+        weatherData={currentData?.[0].swellHeight.data}
+        extraData={currentData?.[0].swellHeight.unit}
+        className={currentData?.[0].swellHeight.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].swellPeriod.dataLabel}
+        weatherData={currentData?.[0].swellPeriod.data}
+        extraData={currentData?.[0].swellPeriod.unit}
+        className={currentData?.[0].swellPeriod.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].windWaveHeight.dataLabel}
+        weatherData={currentData?.[0].windWaveHeight.data}
+        extraData={currentData?.[0].windWaveHeight.unit}
+        className={currentData?.[0].windWaveHeight.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].windWavePeriod.dataLabel}
+        weatherData={currentData?.[0].windWavePeriod.data}
+        extraData={currentData?.[0].windWavePeriod.unit}
+        className={currentData?.[0].windWavePeriod.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].swellDirection.dataLabel}
+        weatherData={currentData?.[0].swellDirection.data}
+        extraData={currentData?.[0].swellDirection.unit}
+        className={currentData?.[0].swellDirection.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].windWaveDirection.dataLabel}
+        weatherData={currentData?.[0].windWaveDirection.data}
+        extraData={currentData?.[0].windWaveDirection.unit}
+        className={currentData?.[0].windWaveDirection.className}
+      />
+      <WeatherCard
+        weatherDataLabel={currentData?.[0].steepness.dataLabel}
+        weatherData={
+          currentData?.[0].steepness.data === 'N/A' ? 'FLAT' : currentData?.[0].steepness.data
```

Comment:
I could just as easily make in NONE

Reply:
Yeah. I don't have a strong view on what label or approach that you should use, but N/A does not mean flat in my mind.

Random suggestions:
- Feel free to be creative with it by flexing your CSS wizardry with icons, animations or whatever comes to mind to indicate that there's no data for that time period. 
- If you want to include some usable data, consider incorporating a statistical average based on trends with the data that you do have to extrapolate a possible range (i.e., wave ranges that surf report sites show).

Reply:
Those are just ideas and not in scope of refactoring.

Workflow Tip:
- It's good practice to separate different types of work to different branches and PRs (as a general rule of thumb, you can think of one branch === one PR).
- So, if you agree with needing to make the change:
  - I advise making minor changes like replacing 'FLAT' with 'N/A', 'None', or '-' (as your permanent solution or as a placeholder so you still have a functioning and accurate page).
   - if you plan on making big changes to the CSS and/or logic, you should do it after this refactor in a separate branch/PR. 
- If you don't agree with the change, then you can disregard all of the above. But, I wanted to share this workflow tip so you have a better understanding of how GitHub and `git` is typically used together in the workplace. `git` is a really powerful tool and it's a game changer. Break down your tasks into branches. The smaller your breakdown (i.e., the smaller your PRs) the easier it will be to catch and fix bugs... and the easier it will be to undo breaking changes too.

As always, let me know if you have any questions. There's a lot to `git` with `git`. :)